tdrs-frontend/README.md

```diff
@@ -37,6 +37,48 @@ CONTAINER ID        IMAGE                        COMMAND                  CREATE
 63f6da197629        tdrs-frontend_tdp-frontend   "/docker-entrypoint.…"   4 seconds ago       Up 3 seconds                      0.0.0.0:3000->80/tcp     tdp-ui
 ```
 
+----
+### Environment Variables
+The TDP frontend utilizes built in Create React App (CRA) handling for environment variables which leverages multiple `.env` files:
+* `.env`: Default.
+* `.env.local`: Local overrides. This file is loaded for all environments except test.
+* `.env.development`, `.env.test`, `.env.production`: Environment-specific settings.
+* `.env.development.local`, `.env.test.local`, `.env.production.local`: Local overrides of environment-specific settings.
+
+All of these `.env` files can be checked in to source control, with the exception of `.local` overrides and provided that they contain no *secrets*.
+
+The order of inheritance for env files depends on how the application was built/launched.
+
+#### Docker
+Currently the Dockerfile.local utilizes `yarn build` and serves the React app over nginx. Due to this React assigns `NODE_ENV=production` and uses this order of inheritance:
```

Comment:
This may be important to bring up with 18F, due to the way the dockerfile deploys the app React acts as if its in a `production` environment. This doesn't technically affect anything at the moment, but if we were to commit a `.env.production` file with variables specific to production the `.env.local` file would need to be present for all local devs in order to prevent environment issues.

Reply:
@jorgegonzalez opened https://github.com/raft-tech/TANF-app/issues/691 to address this issue

Reply:
Thank you for spotting this and looking ahead to it!